WeakAuras/Prototypes.lua

```diff
@@ -491,13 +491,14 @@ end
 function WeakAuras.CheckPvpTalentByIndex(index)
   if (index <= 3) then
     local talentSlotInfo = C_SpecializationInfo.GetPvpTalentSlotInfo(1);
+    if not talentSlotInfo return false
     local checkTalentId = talentSlotInfo.availableTalentIDs[index];
     return talentSlotInfo.selectedTalentID == checkTalentId;
   else
     local checkTalentId = C_SpecializationInfo.GetPvpTalentSlotInfo(2).availableTalentIDs[index - 3];
```

Comment:
The error message seems to point to line 497. Which is this line. Seems GetPvpTalentSlotInfo(2) returned nil here.

Reply:
Additionally, the added line is malformed - it should be `if not talentSlotInfo then return false end`.

Reply:
i sent a patch to fix this